### ISSUE TITLE
Departure times are not adjusting with Dynamic Type

### ISSUE THREAD

**Issue by aaronbrethorst** [OneBusAway/onebusaway-ios#506]
> The estimated time of arrival on the new OneBusA app is green on white and much smaller text size so makes it harder for users with astigmatism and farsightedness like me to read.
> 
> I’ve included another app screenshot which uses black on white and slightly bolder text so easy to read.

![image1](https://user-images.githubusercontent.com/2254/131268154-031823b4-f220-4dbf-97ba-37c03b37b4dc.png)

![image2](https://user-images.githubusercontent.com/2254/131268156-8f742437-b12a-4866-a5bf-186e9c813050.jpeg)

**Comment by ualch9** [comment]
As a temporary workaround, the user can set a larger system text size, which will change the time text into a badge. 
If the user is running iOS 15, they can set a larger text size for just OneBusAway (https://www.theverge.com/22580423/ios-15-iphone-text-size-how-to-control-center).
